include/unifex/reduce_stream.hpp

```diff
@@ -309,22 +315,23 @@ struct _sender<StreamSender, State, ReducerFunc>::type {
 
   template <typename Receiver>
   using operation = operation<StreamSender, State, ReducerFunc, Receiver>;
-
   template <typename Receiver>
-  operation<Receiver> connect(Receiver&& receiver) && {
-    return operation<Receiver>{
-        (StreamSender &&) stream_,
-        (State &&) initialState_,
-        (ReducerFunc &&) reducer_,
-        (Receiver &&) receiver};
-  }
-
+  using next_receiver = next_receiver<StreamSender, State, ReducerFunc, Receiver>;
+  template <typename Receiver>
+  using error_cleanup_receiver = error_cleanup_receiver<StreamSender, State, ReducerFunc, Receiver>;
   template <typename Receiver>
-  operation<Receiver> connect(Receiver&& receiver) const& {
+  using done_cleanup_receiver = done_cleanup_receiver<StreamSender, State, ReducerFunc, Receiver>;
+
+  template (typename Self, typename Receiver)
+      (requires same_as<remove_cvref_t<Self>, type> AND
+          //sender_to<next_sender_t<StreamSender>, next_receiver<Receiver>> AND
```

Comment:
Why is this commented out?
Can you leave a note in the code to explain why this constraint is not being checked?

Reply:
I commented it out because it is causing an error in never_stream_cancellation.cpp with concepts emulation. I forgot I commented it out. The problem remains and needs investigation. I'll dig some more.